zb-db/src/main/java/io/zeebe/db/ZeebeDb.java

```diff
@@ -61,4 +62,25 @@
    * @return {@code true} if the column is empty, otherwise {@code false}
    */
   boolean isEmpty(ColumnFamilyType column, TransactionContext context);
+
+  /**
+   * This method iterates over all entries for a given column family and presents each entry to the
+   * consumer
+   *
+   * <p><strong>Hint</strong> This method should only be used in tests
+   *
+   * @param columnFamily the enum instance of the column family
+   * @param context the context that is used to access the database
+   * @param keyInstance this instance defines the type of the column family key type
+   * @param valueInstance this instance defines the type of the column family value type
+   * @param visitor the visitor that will be called for each entry
+   * @param <KeyType> the key type of the column family
+   * @param <ValueType> the value type of the column family
+   */
+  <KeyType extends DbKey, ValueType extends DbValue> void forEach(
+      final ColumnFamilyType columnFamily,
+      final TransactionContext context,
+      final KeyType keyInstance,
+      final ValueType valueInstance,
+      final BiConsumer<KeyType, ValueType> visitor);
```

Comment:
As I wrote above I would suggest you do this via the column family, especially because it is only used for tests so I would rather not add this here. Otherwise I would expect that you write test for this additional API method.